server/server.test.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+const serverRoutes = require("./routes/rpaFramework"); 
+const request = require("supertest");
+const DBManager = require('./utils/TestingUtils/MongoDbManager');
+const DbTestingMatchers = require('./utils/TestingUtils/DbTestingMatchers');
+
+describe('Test DB-request-routes', () => {
+    const app = express();
+    app.use('/rpa-framework', serverRoutes);
+    const dbman = new DBManager();
+
+    beforeAll(() => dbman.start());
+    afterAll(() => dbman.stop());
+
+    it("attemp to get applications", async () => {
```

Comment:
as far as I understood it, the text should be written in 3. pers sing. because we refer to the test as "it". Therefore the test description should be _attempts to get application_

Not sure about this

Reply:
React Testing Library also uses the test keyword instead of it. Maybe we should change that too. See the [examples](https://testing-library.com/docs/react-testing-library/example-intro/)

Reply:
In my understanding the tests should be described as _this test(it) will try to_ get objects.
But I agree, the attempt should go